plugins/CMakeLists.txt

```
@@ -10,7 +10,7 @@ endif()
 set_property(GLOBAL APPEND PROPERTY OBS_FEATURES_ENABLED "Plugin Support")
 
 macro(check_obs_browser)
-  if((OS_WINDOWS AND CMAKE_VS_PLATFORM_NAME MATCHES "(Win32|x64)") OR OS_MACOS OR OS_LINUX)
+  if(OS_WINDOWS OR OS_MACOS OR OS_LINUX)
```

To be fully correct this should actually be changed into
```suggestion
  if((OS_WINDOWS AND CMAKE_VS_PLATFORM_NAME MATCHES "(ARM64|x64)") OR OS_MACOS OR OS_LINUX)
```
as only those two architectures generate a working OBS-Studio build and Win32 just "exists" to build the virtual camera and graphics helpers.

---

Good point, fixed.